simulator/fetch/fetch.cpp

```diff
@@ -11,6 +11,11 @@ namespace config {
     static Value<std::string> bp_mode = { "bp-mode", "dynamic_two_bit", "branch prediction mode"};
     static Value<uint32> bp_size = { "bp-size", 128, "BTB size in entries"};
     static Value<uint32> bp_ways = { "bp-ways", 16, "number of ways in BTB"};
+
+    /* Cache parameters */
+    static Value<uint32> size_in_bytes = {"size,S", 2048, "cache size in bytes"};
```

Comment:
Even worse. How should user understand that `-S 4096 -w 8` options configure instruction cache and not data cache, level 2 cache, or bypass width? BP options above are clearly described.

I expected:
```c++
/* ... */ instruction_cache_size =  { "icache_size", 2048, "Size of instruction level 1 cache (in bytes)" };
/* ... */ instruction_cache_ways = { "icache_ways", 4, "Amount of ways in instruction level 1 cache" };
/* ... */ instruction_cache_line_size = { "icache_line_size", 64, "Line size of instruction level 1 cache (in bytes)" }
```

Reply:
Sorry, my mistake...just a minute, I'll fix it

Reply:
It is not a mistake, it is untidiness. Please keep in mind that results of your work will be used or even modified by others or by yourself later. Try to keep your code, comments, variable names clean and informative on your own. It is better to spent 5 minutes now (and we can afford this!) than an hour of debugging later.